lib/components/narrative/line-itin/place-row.js

```diff
@@ -156,14 +170,23 @@ class RentedVehicleLeg extends PureComponent {
       )
     }
 
-    if (leg.rentedVehicleData) {
+    if (leg.rentedVehicle || leg.rentedBike || leg.rentedCar) {
+      // console.log(leg.from.networks, configCompanies)
+      const companies = leg.from.networks.map(n => getCompanyForNetwork(n, configCompanies))
+      const companyLabel = companies.map(co => co.label).join('/')
+      const modeString = getModeStringForCompany(companies[0])
```

Comment:
Since a company can offer multiple modes, this may not be future-proof. Can't we just use the mode of the leg to get the label?